### PR TITLE
Awesome books project

### PR DESCRIPTION
**Pull request by agneta1991**
## Awesome Books project:

**pair-programming done by:**
***Agneta Pupienyte & Mohammed Roshdy***

- created an array to keep the list of books.
- created a function to add a new book to the collection, with title and author.
- created a function to remove a book from the collection
- all books saved in the collection are displayed in the top part of the page.
- buttons enabled to add and remove books from the page & from the object array

- all user inputs saved as data collection in local storage

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @agneta1991 and @MohamedRoshdy2021,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### - Kindly add a live demo link in the live demo section or remove the section entirely
### - You have not set up linters for this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: javascript.js [hunk 1]
```diff
@@ -1 +1,57 @@
-console.log ('im working :) ')
\ No newline at end of file
+const books = [];
```
Comment (ShadyShawkat) [inline javascript.js]: - [ ] Please make sure the list is preserved in the local storage and reloaded into the page when it starts, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Make%20sure%20that%20data%20is%20preserved%20in%20the%20browser%27s%20memory%20by%20using%20localStorage.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @agneta1991, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Efiamotu-1 — COMMENTED** [review]
Hi @agneta1991 & @MohamedRoshdy2021 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 

You have submitted a project, that has not setup linters. please follow the instructions in this [link](https://github.com/microverseinc/linters-config/tree/master/javascript) to set up linters for this project :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by bobb-Rob — COMMENTED** [review]
Hi @agneta1991 & @MohamedRoshdy2021 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has not setup linters. please follow the instructions in this [link](https://github.com/microverseinc/linters-config/tree/master/javascript) to set up linters for this project 👍

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: .github/workflows/linter.yml
Comment (AbelG101) [inline .github/workflows/linter.yml:1]: - [ ] [OPTIONAL] The previous reviewer forgot to mention this but In the future please make sure your branch name is the name of a feature or an action for example, you can name this branch as `add_plain_JS_with_objects` or `add_remove_feature` this better describes the changes on the branch when compared with `first-feature` which is a bit too vague. ☺

**Review by AbelG101 — APPROVED** [review]
# Hi @agneta1991 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Books are properly added & removed ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linter.yml
Comment (MohamedRoshdy2021) [reply .github/workflows/linter.yml:1]: yeah you are right i just  forgot💯

**Pull request merged**